gradle/versions.gradle

```diff
@@ -29,11 +29,13 @@ dependencyManagement {
 
     dependency 'io.rest-assured:rest-assured:4.2.0'
 
-    dependency 'io.vertx:vertx-codegen:3.8.2'
-    dependency 'io.vertx:vertx-core:3.8.2'
-    dependency 'io.vertx:vertx-unit:3.8.2'
-    dependency 'io.vertx:vertx-web-client:3.8.2'
-    dependency 'io.vertx:vertx-web:3.8.2'
+    dependencySet(group: 'io.vertx', version: '3.9.8') {
```

Comment:
Looks like latest version is 3.9.9 https://vertx.io/blog/eclipse-vert-x-3-9-9/ can we update to that

Reply:
@usmansaleem flagged this duration our conv yday to keep the version inline with web3signer... @usmansaleem is there specific reason for keeping them the same?